### DISCUSSION TITLE
ETA for initial merge of Swift bindings?

### DISCUSSION THREAD

**Discussion by douglasdew** [ml-explore/mlx#491] [General]
Does anybody have an ETA for the initial merge of the Swift bindings?

**Comment by awni** [comment]
Hey! @davidkoski can say more, he is driving the initial implementation. Maybe 2 weeks if we don’t run into any problems, but hard to be certain.

**Reply by davidkoski** [reply]
Yes, something like that -- good progress on it so far.

**Reply by douglasdew** [reply]
That is fantastic news! I had been expecting something more along the lines of mid-year.

I am very much looking forward to seeing how you've solved problems such as providing support for array slicing. I hope that the Swift bindings will be as easy to use as the Python bindings.
